src/runtime/policy/tests/policy-test.ts

```diff
@@ -435,28 +435,74 @@ policy MyPolicy {
     const policy = manifest.policies[0];
     const schema = policy.targets[0].getMaxReadSchema();
     const expectedSchemas = (await Manifest.parse(`
-      schema Address
-        number: Number
-        street: Text
-        city: Text
-        country: Text
-
-      schema Name
-        last: Text
-
       schema Person
         name: inline Name {last: Text}
-        addresses: List<inline Address {number, street, city}>
+        addresses: List<inline Address {number: Number, street: Text, city: Text}>
       `)).schemas;
     assert.deepEqual(schema, expectedSchemas['Person']);
   });
 
+  it(
+    'Deeply nested schemas are also restricted according to policy', async () => {
```

Comment:
FYI: Very unusual to have \n after `it(`

Reply:
Good idea. Will fix it.